tests/test_guid_based_recommender.py

```diff
@@ -119,31 +116,51 @@ def test_rownorm_sumrownorm(default_ctx, MOCK_DATA):
 
     That gives a final expected weight for guid-1 to be: 0.403591682
     """
-    expected = ('guid-1', 0.403591682)
-    assert compare_actual_expected((actual[1], expected))
+    expected = 0.403591682
+    actual = float(actual[1][1][:-11])
+    assert expected == pytest.approx(actual, rel=1e-3)
 
 
 @mock_s3
-def test_rowsum_recommender(default_ctx, MOCK_DATA):
+def test_rowsum_recommender(default_ctx, MOCK_DATA, MOCK_GUID_RANKING):
     EXPECTED_RESULTS = RESULTS['row_sum']
-    install_mock_data(MOCK_DATA)
+    install_mock_data(MOCK_DATA, MOCK_GUID_RANKING)
 
     recommender = GuidBasedRecommender(default_ctx)
     guid = "guid-2"
 
     actual = recommender.recommend({'guid': guid, 'normalize': 'row_sum'})
     assert 4 == len(actual)
-    assert compare_actual_expected((('guid-1', 50/155), actual[0]))
+
+    expected_val = 50/155
+    actual_val = float(actual[0][1][:-11])
+    assert expected_val == pytest.approx(actual_val, rel=1e-3)
+
     assert actual == EXPECTED_RESULTS
 
 
 @mock_s3
-def test_guidception(default_ctx, MOCK_DATA):
+def test_guidception(default_ctx, MOCK_DATA, MOCK_GUID_RANKING):
     EXPECTED_RESULTS = RESULTS['guidception']
-    install_mock_data(MOCK_DATA)
+    install_mock_data(MOCK_DATA, MOCK_GUID_RANKING)
 
     recommender = GuidBasedRecommender(default_ctx)
     guid = "guid-2"
 
     actual = recommender.recommend({'guid': guid, 'normalize': 'guidception'})
     assert actual == EXPECTED_RESULTS
+
+
+@mock_s3
+def test_rownorm_sum_tiebreak(default_ctx, TIE_MOCK_DATA, MOCK_GUID_RANKING):
+    EXPECTED_RESULTS = RESULTS['rownorm_sum_tiebreak']
+    install_mock_data(TIE_MOCK_DATA, MOCK_GUID_RANKING)
+
+    recommender = GuidBasedRecommender(default_ctx)
+    guid = "guid-2"
+
+    actual = recommender.recommend({'guid': guid, 'normalize': 'rownorm_sum'})
```

Comment:
here the normalization strategy is being explicitly passed as a parameter. However, 'rownorm_sum' is the default normalization strategy. Can we add an additional assertion in this test that 

`recommender.recommend({'guid': guid})  == EXPECTED_RESULTS`

in addition to the currently used assertions?

Reply:
Oh yeah, I actually test that over here: https://github.com/mozilla/taar-lite/pull/17/files/032b4a547683dd928d3c14c1decfabae43e0150b#diff-c791252c63cf945e8e08b32e0689432fR100